packages/fx-core/src/common/samples-config.json

```diff
@@ -207,6 +207,16 @@
             "configuration": "Ready for debug",
             "suggested": false
         },
+        {
+            "id": "teams-videoapp-sample",
+            "title": "Video Filter App Sample",
+            "shortDescription": "A minimal sample app with basic video processing capabilities",
+            "fullDescription": "A minimal sample app with basic video processing capabilities.",
+            "tags": ["Video filter", "Video app", "AR", "Video", "Video Stream", "Video Extensibility"],
```

Comment:
Are tags too many? Looks to me "Video Filter" and "JS" is enough.

Reply:
Updated.